### ISSUE TITLE
CognitiveServicesManagementClient create() account should return a LROPoller 

### ISSUE THREAD

**Issue by kristapratico** [Azure/azure-sdk-for-python#7476]
In the accounts operations, the create() operation can take quite a bit of time to actually create the cognitive services account. This should return an LROPoller and allow the user to poll for the result.

**Comment by Caynosadler** [comment]
Hi @kristapratico is this issue still open, can i take a look at it ?

**Comment by Caynosadler** [comment]
@kristapratico 

An LROPoller is being used currently for the DataLake accounts create [operation.](https://github.com/Azure/azure-sdk-for-python/blob/ab599e7c6c371543e48d210c4a11f163951d83d4/sdk/datalake/azure-mgmt-datalake-store/azure/mgmt/datalake/store/operations/accounts_operations.py) I'll be replicating the same approach for CognitiveServicesManagementClient. 

Kindly advice if it's okay to proceed with this approach.

**Comment by Caynosadler** [comment]
` raw_result = self._create_initial(
            resource_group_name=resource_group_name,
            account_name=account_name,
            parameters=parameters,
            custom_headers=custom_headers,
            raw=True,
            **operation_config
        )

        def get_long_running_output(response):
            deserialized = self._deserialize('DataLakeStoreAccount', response)

            if raw:
                client_raw_response = ClientRawResponse(deserialized, response)
                return client_raw_response

            return deserialized

        lro_delay = operation_config.get(
            'long_running_operation_timeout',
            self.config.long_running_operation_timeout)
        if polling is True: polling_method = ARMPolling(lro_delay, **operation_config)
        elif polling is False: polling_method = NoPolling()
        else: polling_method = polling
        return LROPoller(self._client, raw_result, get_long_running_output, polling_method)
    create.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataLakeStore/accounts/{accountName}'}`


Snippet from the DataLake create operation

**Comment by kristapratico** [comment]
Hey @Caynosadler, I looked into this a bit and realized that the above LROPoller code snippet is actually auto-generated from the swagger found in this [repo](https://github.com/Azure/azure-rest-api-specs). 

I believe your approach is correct, but if we made any changes to this file it would just be overwritten when the code is re-generated. Since the change needs to be made in the swagger, I will follow up with this issue so that we can generate it. Thanks for your help :)
